### ISSUE TITLE
LaTeX code in documentation not rendered

### ISSUE THREAD

**Issue by CsatiZoltan** [mathLab/PyGeM#244]
Formulae are not rendered on the .github.io page (e.g. on https://mathlab.github.io/PyGeM/ffdcad.html), neither in Firefox, nor in a Chrome-based browser. Strangely, when I cloned the repository and built the docs by myself with `make html`, it was shown properly. What could be the reason?

**Comment by kshitij-maths** [comment]
I rechecked the build locally, and the formulas are rendering correctly. However, the live website is not updating because the GitHub Action responsible for deployment is not running.

In my view, the reasons are as follows:

1. **Wrong Trigger (main reason):** It currently only runs on tags (releases), so merges to master do not trigger a rebuild of the website.
2. **Deprecated Runner:** It uses ubuntu-18.04, which GitHub has removed. This may cause the job to fail even if triggered.
3. **Old Environment (optional):** It uses Python 3.7. Perhaps not affecting this particular task but it's better to update this inline with the recent updates in the package.

I will open a new Pull Request to modernize `sphinx-build.yml` to fix the aforesaid issues.

**Comment by ndem0** [comment]
You should be able to push a new tag!
BTW, I did it right now, the online documentation is updated online, but `CAD.FFD` and `CAD.CADDeformation` are not shown at all.
So please do the needful! And please double check locally!

**Comment by kshitij-maths** [comment]
Do you mean the following links?

- `CAD.FFD`: https://mathlab.github.io/PyGeM/ffdcad.html
- `CAD.CADDeformation`: https://mathlab.github.io/PyGeM/cad_deformation.html

They were already available locally. Also I checked online they are live now. Please see the attached images and let me know if it is okay or I'm missing anything.

<img width="1024" height="640" alt="Image" src="https://github.com/user-attachments/assets/798ab7a7-a20d-42e1-82c4-dc552b407e27" />
<img width="1024" height="640" alt="Image" src="https://github.com/user-attachments/assets/8d2740e9-9013-4354-af6a-6bc6395f8201" />

**Comment by ndem0** [comment]
Yes, you're right, now I can see everything! I don't know why initially those empty were empty at all. My bad, sorry! I'll close the issue!
